deployments/image/configmap/Dockerfile

```diff
@@ -35,4 +35,3 @@ RUN chmod +x /generate-signed-ca.sh
 ADD k8s_yunikorn_scheduler /k8s_yunikorn_scheduler
 WORKDIR /
 ENTRYPOINT ["/k8s_yunikorn_scheduler"]
-CMD ["-logEncoding=console", "-logLevel=-1", "-clusterId=mycluster", "-clusterVersion=latest"]
```

Comment:
Will this be enough? I thought we also need to specify ENVs in the docker file?
If these arguments are removed, how the binary `k8s_yunikorn_scheduler` receive CLI arguments?

Should we do something like
```
ENV LOG_ENCODING "console"
ENV LOG_LEVEL "-1"
ENV ...

CMD ["-logEncoding=${LOG_ENCODING}", "-logLevel=${LOG_LEVEL}", "-clusterId=${CLUSRTER_ID}", "-clusterVersion=${CLUSTER_VERSION}"]
```
just a thought, not tested. Found more info: https://stackoverflow.com/questions/40454470/how-can-i-use-a-variable-inside-a-dockerfile-cmd

Reply:
You are right! I checked only if the environment variables are set. Unfortunately the solution proposed by you is not working. In order to pass the variable I had to execute a shell as described here: https://docs.docker.com/engine/reference/builder/#cmd

After this solution I tested it by checking in the logs if the passed values are used(both the ones from the docker file and from the .yaml file as well) instead of the default ones.